ext/duckdb/result.c

```diff
@@ -65,11 +65,7 @@ static VALUE to_ruby_obj_string_from_blob(duckdb_result *result, idx_t col_idx,
     str = rb_str_new(bval.data, bval.size);
 
     if (bval.data) {
-#ifdef HAVE_DUCKDB_FREE
-        duckdb_free(bval.data);
```

Comment:
Don't remove this line.

Reply:
Replaced free() with duckdb_free() in line 69.